src/config/config.cpp

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <fstream>
+#include <iostream>
+#include <unordered_map>
+
+#include <Poco/JSON/Object.h>
+#include <Poco/JSON/Parser.h>
+
+#include "config.hpp"
+
+/***********************************************************************************************************************
+ * Types
+ **********************************************************************************************************************/
+
+class CaseInsensitiveObjectWrapper {
+public:
+    CaseInsensitiveObjectWrapper(const Poco::JSON::Object::Ptr& object)
+        : mObject(object)
+    {
+        for (const auto& pair : *object) {
+            mKeyMap.emplace(ToLowercase(pair.first), pair.first);
+        }
+    }
+
+    bool has(const std::string& key) const
```

Comment:
Method names should start from capitl letter.

Reply:
Done